<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlephZero Ethereum Client</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="net">
        <property name="text">
         <string>Net</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="port">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>32767</number>
        </property>
        <property name="value">
         <number>30303</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="value">
        <property name="suffix">
         <string> wei</string>
        </property>
        <property name="maximum">
         <number>430000000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fee">
        <property name="suffix">
         <string>)</string>
        </property>
        <property name="prefix">
         <string>(fee </string>
        </property>
        <property name="maximum">
         <number>430000000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destination"/>
      </item>
      <item>
       <widget class="QToolButton" name="send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="create">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="mineProgress" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="mine">
        <property name="text">
         <string>Mine</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="peers"/>
     <widget class="QListWidget" name="accounts">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="transactionQueue"/>
     <widget class="QListWidget" name="transactions"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="balance">
        <property name="text">
         <string>0 wei</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="address">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blockChain">
        <property name="text">
         <string>1 block</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="peerCount">
        <property name="text">
         <string>0 peers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
